boards/t480-hotp-maximized/t480-hotp-maximized.config

```diff
@@ -77,13 +76,11 @@ export CONFIG_TPM2_TOOLS=y
 export CONFIG_PRIMARY_KEY_TYPE=ecc
 #TPM1 requirements
 #export CONFIG_TPM=y
-#Enable DEBUG output, debug output probably a good idea for first tests TODO:remove prior of merge
-export CONFIG_DEBUG_OUTPUT=y
+export CONFIG_DEBUG_OUTPUT=n
 export CONFIG_ENABLE_FUNCTION_TRACING_OUTPUT=y
```

Comment:
@gaspar-ilom should also be =n as per all other boards.

Reply:
Fixed under https://github.com/linuxboot/heads/pull/1906/commits/d9f4cbcfc348d1e1fc50ac27241811f3b0ef9919